EKF/ekf.cpp

```diff
@@ -389,6 +389,8 @@ bool Ekf::initialiseFilter(void)
 		}
 	}
 
+	// warnx("_params.fusion_mode & MASK_USE_EVPOS %d", (int)(_params.fusion_mode & MASK_USE_EVPOS));
```

Comment:
Can you please replace these with print statements or something else that does not crete a dependance on the PX4 library. We are trying to keep ecl platform independant